.circleci/src/config/jobs/verify-ci-should-run.yml

```diff
@@ -0,0 +1,49 @@
+resource_class: small
```

Comment:
This strangely maxes out its resources checking out code. I wonder if this could be sped up with more. (I realize you didn't change this from before). https://app.circleci.com/pipelines/github/cypress-io/cypress/74784/workflows/2d0ba1d8-892a-4aee-9cac-b0d07ffd0700/jobs/3142881/resources